src/Sylius/Behat/Service/NotificationValidatorInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Sylius\Behat\Service;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+interface NotificationValidatorInterface
+{
+    /**
+     * @param string $resource
```

Comment:
The exception constructor may receive the searched notification and all existing notifications, so that we can display a really nice message for the developer, like `Expected *success* notification with "lorem ipsum" message, got *error* "abc def", *success* "ghi jpg"`.